.github/workflows/check-pr-to-release.yml

```diff
@@ -38,3 +38,30 @@ jobs:
                     echo "Base branch has commits since merge base"
                     exit 1
                   fi
+
+    check_source_branch:
+        name: Check source branch
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Assert source branch is RC branch
+              if: ${{ !startsWith(github.head_ref, 'rc/') }}
+              run: exit 1
+
+    check_release_version:
+        name: Check release version
+        needs: check_source_branch
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Parse version
+              id: version
+              uses: ./.github/parse-version
```

Comment:
does this require checking out the repo first?

Reply:
using my intuition and added a checkout step before this, but just for the necessary packge.json file: https://github.com/digidem/release-automations-expo/pull/7/commits/a9871aae8c78739f42252dd32a718fe6f20aafb4

Reply:
👍 Good idea.